src/tox/config.py

```diff
@@ -1018,20 +1018,17 @@ def __init__(self, config, ini_path, ini_data):  # noqa
         if override:
             for name in config.indexserver:
                 config.indexserver[name] = IndexServerConfig(name, override)
-        unique_id = str(uuid.uuid4())
 
         reader.addsubstitutions(toxworkdir=config.toxworkdir)
         config.distdir = reader.getpath("distdir", "{toxworkdir}/dist")
-        self._make_thread_safe_path(config, "distdir", unique_id)
 
         reader.addsubstitutions(distdir=config.distdir)
         config.distshare = reader.getpath("distshare", dist_share_default)
-        self._make_thread_safe_path(config, "distshare", unique_id)
+        config.temp_dir = reader.getpath("temp_dir", "{toxworkdir}/.tmp")
```

Comment:
@gaborbernat if I'm reading this right, you forgot to add it to substitutions

Reply:
This is meant as an internal implementation detail folder, not meant for public usage.

Reply:
Ah, the docs suggest otherwise...

Reply:
Well, I guess now it's supported.

Reply:
Hyrum's Law, huh?